package.json

```diff
@@ -43,7 +43,16 @@
     "@babel/register": "^7.10.5",
     "@hapi/code": "^8.0.1",
     "@hapi/lab": "^24.0.0",
+    "@rpii/wdio-html-reporter": "^6.1.1",
+    "@wdio/cli": "^6.6.0",
+    "@wdio/cucumber-framework": "^6.6.0",
+    "@wdio/local-runner": "^6.6.0",
+    "@wdio/mocha-framework": "^6.6.0",
+    "@wdio/selenium-standalone-service": "^6.6.0",
+    "@wdio/spec-reporter": "^6.6.0",
+    "@wdio/sync": "^6.6.0",
```

Comment:
could you move these into test/package.json please. When you're thinking about running these in CI, we can set up a yarn workspace for `test`

Reply:
I can indeed.